x-pack/plugins/security_solution/scripts/endpoint/common/fleet_services.ts

```diff
@@ -209,9 +215,70 @@ export const waitForHostToEnroll = async (
     );
   }
 
-  log.debug(`Host [${hostname}] has been enrolled with fleet`);
+  log.info(`Host [${hostname}] has been enrolled with fleet`);
```

Comment:
I would suggest changing this back to `.debug()`

Reply:
Changed